src/PrettyOrdering.jl

```diff
@@ -0,0 +1,46 @@
+pretty_lt(x, y) = false
+
+pretty_lt(x::Integer, y::Integer) = isless(x, y)
+
+pretty_lt(x::Rational{<:Integer}, y::Rational{<:Integer}) = isless(x, y)
+
+# not sure about the default
+pretty_eq(x, y) = (x == y)
+
+function pretty_lt(x::T, y::T) where {T <: Union{Vector, Tuple}}
+  return pretty_lt_it(x, y)
```

Comment:
```suggestion
  return pretty_lt_lex(x, y)
```
at least this is I assume you want here